src/app/global.css

```
@@ -804,6 +805,13 @@ https://velog.io/@oneook/tailwindcss-4.0-%EB%AC%B4%EC%97%87%EC%9D%B4-%EB%8B%AC%E
   letter-spacing: 0;
 }
 
+@utility font-designer-22m {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)